server/builtin/app.go

```diff
@@ -95,7 +95,7 @@ func NewBuiltinApp(conf config.Service, proxy proxy.Service, httpOut httpout.Ser
 func Manifest(conf config.Config) apps.Manifest {
 	return apps.Manifest{
 		AppID:       AppID,
-		Version:     apps.AppVersion(conf.BuildConfig.BuildHashShort),
+		Version:     apps.AppVersion(conf.PluginManifest.Version + "-" + conf.BuildHashShort),
```

Comment:
This may not be a valid version strings in some contexts, but appears to work. I can take this change out if controversial.

Reply:
I'm hesitant to add that as it might break at some point and that might be hard to debug.

Also: What is the value of building the version like this?